tests/resources/promise-rejection-events.js

```diff
@@ -586,6 +673,121 @@ async_test(function(t) {
   Promise.reject(e);
 }, 'mutationObserverMicrotask vs. postMessageTask ordering is not disturbed inside unhandledrejection events');
 
+// For the next two see https://github.com/domenic/unhandled-rejections-browser-spec/issues/2#issuecomment-121121695
+// and the following comments.
+
+async_test(function(t) {
+  var sequenceOfEvents = [];
+
+  addEventListener('unhandledrejection', l);
+  ensureCleanup(t, l);
+
+  var p1 = Promise.reject();
+  var p2;
+  postMessageTask(function() {
```

Comment:
on workers, the MessageChannel port postMessage thing bounces to the main thread, and as such there are no ordering guarantees wrt to the promise events